how-to/observability.rst

```diff
@@ -5,23 +5,8 @@ Observability
 
 In Ubuntu, it is recommended to use the
 `Canonical Observability Stack <https://documentation.ubuntu.com/observability/>`_
-to monitor your infrastructure.
+to monitor your infrastructure. For more information, see the following links:
 
-However, you can also use the classic Logging, Monitoring and Alerting (LMA)
-stack.
-
-.. toctree::
-   :titlesonly:
-
-   Set up your LMA stack <observability/set-up-your-lma-stack>
```

Comment:
I'm happy for us to take away the LMA stack, since this has been fully superseded by COS. However, I would ask you to also delete the lma stack page in question, otherwise the documentation will fail to build because of the page existing without being in a toctree.